connections/walkietalkie/app/src/main/java/com/google/location/nearby/apps/walkietalkie/ConnectionsActivity.java

```diff
@@ -92,7 +94,8 @@
   private final ConnectionLifecycleCallback mConnectionLifecycleCallback =
       new ConnectionLifecycleCallback() {
         @Override
-        public void onConnectionInitiated(String endpointId, ConnectionInfo connectionInfo) {
+        public void onConnectionInitiated(
+            @NonNull String endpointId, ConnectionInfo connectionInfo) {
```

Comment:
Sorry to be picky, but would you also mind reverting the @ NonNull annotation additions? Code in google's repository assumes @ NonNull is the default when not specified (it's a known and slightly irritating issue that this conflicts with Android Studio's default behavior, a battle that continues to rage on in some corner of the Googleverse, ho hum).

Reply:
Sure, no problem

Reply:
I've undone the @ NonNull annotation additions I made. I left the @ NonNull annotations that were already there since I figured they are probably there for good reason. If not, I'd be happy to remove them.